apps/web/src/components/shared/ErrorBoundary.tsx

```
@@ -1,63 +1,41 @@
+          <p className="text-sm text-muted-foreground">
+            An unexpected error occurred. Try reloading the application.
+          </p>
+          {error && (
+            <pre className="text-xs text-red-400 bg-muted/50 rounded-md p-3 max-w-full overflow-auto">
+              {error.message}
+            </pre>
+          )}
+          <button
+            onClick={() => window.location.reload()}
+            className="flex items-center gap-2 px-4 py-2 rounded-md bg-primary text-primary-foreground text-sm hover:bg-primary/90 transition-colors"
+          >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)